README.md

```diff
@@ -16,3 +16,22 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+How would you verify this claim? 
+
+
+I would start by running it through lists of various sizes, both sorted and unsorted. Then, if it truly sorts the lists, I will graph how long it took to sort each list. If its claims are true, then this should be linear. 
+
+
+give a theoretical argument for why X could or could not be correct, based
+on the complexity of the general sorting problem we covered in class.
+
+
+We currently have a few sorting algorithms with the best-case scenario complexity of N, but this is only when they are given a sorted list. The best-case scenario for a random list is a time complexity of O(nlogn), and this is only because they implement the idea of transitivity and a divide-and-conquer design strategy. Unless we find a third magical immutable law of numbers that allows us to intuit the condition of a portion of the array, there is no way to have an algorithm with the worst-case scenario of O(n).
```

Comment:
And what do you know about the complexity of the sorting *problem* that is relevant here?